.github/workflows/release.yml

```diff
@@ -47,21 +47,6 @@ jobs:
           # This path is from the root of the repo as needed by the plugin
           files: prime-router/build/test-results/test/**/*.xml
 
-      - name: Run Integration Tests
```

Comment:
Context: release build is not running containers or smoke tests, so integration tests need to be removed.